app.js

```diff
@@ -24,12 +29,18 @@ app.use(methodOverride('_method'));
 app.use(require('stylus').middleware(__dirname + '/public'));
 app.use(express.static(path.join(__dirname, 'public')));
 
+
 if (app.get('env') == 'development') {
 	app.locals.pretty = true;
 }
-
-app.get('/', routes.index);
+app.get('/', function(req, res){
+  var dat = JSON.stringify(data)
+	res.render('index', {
+    data: data,
+    data2: dat
```

Comment:
Confused about this: is it meant to be recursive? The output is `dat` and it gets `dat`?